src/main/java/javax/validation/ConstraintViolationException.java

```diff
@@ -32,7 +34,7 @@ public ConstraintViolationException(String message,
 			this.constraintViolations = null;
 		}
 		else {
-			this.constraintViolations = new HashSet<ConstraintViolation<?>>( constraintViolations );
+			this.constraintViolations = new HashSet<>( constraintViolations );
```

Comment:
I agree that it is a behavioral change and therefore perhaps doesn't belong into this PR. 

However, I also think that nobody is expecting this `Set` to be mutable, so perhaps we should make it immutable AND adjust the javadocs in a separate PR?